cosmicds/components/mc_radiogroup.vue

```diff
@@ -112,29 +112,35 @@ module.exports = {
       iconWrong: 'mdi-alert-circle-outline',
       complete: false,
       tries: 0,
-      score: null
+      score: null,
+      wrongAttempts: 0
     };
   },
   methods: {
-    selectChoice: function(index, send=true) {
+    selectChoice: function(index, forInitialization=false) {
       this.column = index;
       const correct = this.correctAnswers.includes(index);
       const neutral = this.neutralAnswers.includes(index);
-      if (!neutral) {
+      const wrong = !(correct || neutral);
+      if (!forInitialization) {
         this.tries += 1;
+        if (wrong) {
+          this.wrongAttempts += 1;
+        }
       }
       this.complete = correct || (this.correctAnswers.length === 0 && neutral);
       if (this.scoring && this.complete) {
-        this.score = correct ? this.getScore(this.tries) : 0;
+        this.score = correct ? this.getScore(this.wrongAttempts) : 0;
```

Comment:
So I think to @patudom's comment, we would want it to be: 

```suggestion
        this.score = (correct || neutral) ? this.getScore(this.wrongAttempts) : 0;
```

Reply:
@Carifio24, yes, if there is a correct answer and a neutral answer, we want the student to keep trying until they get to the correct answer. If all answers are neutral, any choice should give them 10 points. 

I don't recall if we have questions with some neutral and some wrong (but no correct) answers, but in that case, I would expect that any wrong choices would reduce their points, and choosing the neutral answer would provide them a score and allow them to advance.